src/obfl-specification.html

```diff
@@ -1320,6 +1320,29 @@ <h4 id="L836" class="include">page-number</h4>
 </dl>
 </div>
 
+<h4 id="L840" class="include">metadata</h4>
+
+<ins datetime="20200429T15:00:00">The metadata attribute can be added to any block in order
```

Comment:
> The metadata attribute

It is an element.

> to any block

I would make this a bit more specific/accurate and say that it can be attached to any position in the text within the block.